electric_slide.gemspec

```diff
@@ -23,9 +23,9 @@ Gem::Specification.new do |s|
   s.rubygems_version = "1.2.0"
   s.summary = "Automatic Call Distributor for Adhearsion"
 
-  s.add_runtime_dependency 'adhearsion'
+  s.add_runtime_dependency 'adhearsion', ['>= 2.5.0', '< 3.0.0']
   s.add_runtime_dependency 'countdownlatch'
-  s.add_runtime_dependency 'activesupport'
+  s.add_runtime_dependency 'activesupport', ['<= 3.2.13']
```

Comment:
Just curious - what was the issue here? Is the project using a method that was deprecated after this version of AS?

Reply:
Yes, eager_load! was deprecated after this version of activesupport